web/app.php

```diff
@@ -27,12 +33,11 @@
 // See http://symfony.com/doc/current/book/http_cache.html#symfony-reverse-proxy
 //
 // require_once __DIR__.'/../app/AppCache.php';
-// $kernel = new AppKernel('prod', false);
-// $kernel->loadClassCache();
 // $kernel = new AppCache($kernel);
 // Request::enableHttpMethodParameterOverride();
```

Comment:
But in the backend we use the `DELETE` method (https://github.com/symfony/symfony-demo/blob/master/src/AppBundle/Controller/Admin/BlogController.php#L186)  Can we still ignore this line?